### ISSUE TITLE
[TODO] Allow `config.yaml`urable path inclusion/exclusion

### ISSUE THREAD

**Issue by webknjaz** [sanitizers/chronographer-github-app#11]
This should facilitate projects which may want to only require change fragments if certain files/dirs are changed.

```yaml
paths:
  include: []  # default: include all files
  exclude: []  # changes to these files shouldn't require change fragments
```

**Comment by webknjaz** [comment]
(idea by @abadger)

**Comment by webknjaz** [comment]
Resolved by 05db17c.
